vendor/src/github.com/docker/notary/keystoremanager/import_export.go

```diff
@@ -42,6 +42,39 @@ func (km *KeyStoreManager) ExportRootKey(dest io.Writer, keyID string) error {
 	return err
 }
 
+// ExportRootKeyReencrypt exports the specified root key to an io.Writer in
+// PEM format. The key is reencrypted with a new passphrase.
+func (km *KeyStoreManager) ExportRootKeyReencrypt(dest io.Writer, keyID string, newPassphraseRetriever passphrase.Retriever) error {
```

Comment:
We added this code, but it isn't actually being used by docker. It's only used by notary to export/reencrypt root keys.